packages/block-editor/src/components/block-actions/index.js

```diff
@@ -98,16 +98,10 @@ export default function BlockActions( {
 			return removeBlocks( clientIds, updateSelection );
 		},
 		onInsertBefore() {
-			const clientId = Array.isArray( clientIds )
-				? clientIds[ 0 ]
-				: clientId;
-			insertBeforeBlock( clientId );
```

Comment:
I believe we missed a typo while refactoring in #43846. The previous logic would result in the following error if `clientIds` wasn't array: `Uncaught ReferenceError: clientId is not defined`.

cc @tyxla

Reply:
Great catch, thanks I confirm this had to be `clientIds` in the first place!